framework/src/modules/random/constants.ts

```diff
@@ -21,5 +21,5 @@ export const ADDRESS_LENGTH = 20;
 export const MAX_HASH_COMPUTATION = 10000;
 
 export const defaultConfig = {
-	maxLengthReveals: DEFAULT_MAX_LENGTH_REVEALS,
+	maxLengthReveals: MAX_LENGTH_REVEALS,
```

Comment:
Hmmm it seems like `maxLengthRevealsMainchain` is not used anywhere, because in `RandomModule.init()` we don't specify the interface but use:

```typescript
validator.validate<{ maxLengthReveals: number }>(randomModuleConfig, config);
```

I will remove the unused interface.